### DISCUSSION TITLE
Implementing repo sidebar for blog?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#487] [General]
Is there any reason why you implemented the repo sidebar thing for docs but not for the blog? I could imagine one would use that for the blog, too. I will do that some day :)

**Comment by razonyang** [comment]
The `repo` is just a widget, you can place it in posts's sidebar via [Hooks](https://hbs.razonyang.com/en/docs/hooks/).

```
// /layouts/partials/hooks/sidebar-begin.html
{{ partial "docs/repo" . }}
```
